test/services/rulesService.test.js

```diff
@@ -1,13 +1,70 @@
-
 const rulesService = require('../../server/services/rulesService')
+
+jest.mock(
+  '../../data/rules.json',
+  () => ([
+    {
+      id: 1,
+      description: 'Mock rule 1',
+      enabled: true,
+      facts: []
+    },
+    {
+      id: 2,
+      description: 'Mock rule 2',
+      enabled: false,
+      facts: []
+    },
+    {
+      id: 3,
+      description: 'Mock rule 3',
+      enabled: true,
+      facts: []
+    }
+  ])
+)
+
 describe('rulesService', () => {
-  test('rules service returns an array', async () => {
+  const ruleID1 = 1
+  const ruleID2 = 2
+  const enabled1 = false
+  const enabled2 = true
+
+  afterAll(() => {
+    jest.unmock('../../data/rules.json')
+  })
+
+  test('rules service get returns an array', async () => {
     const rules = await rulesService.get()
     expect(Array.isArray(rules)).toEqual(true)
   })
 
-  test('rules service returns an array where the first element has an id of 1', async () => {
+  test('rules service get returns an array where the first element has an id of 1', async () => {
     const rules = await rulesService.get()
     expect(rules[0]).toMatchObject({ id: 1 })
   })
+
+  test('rules service updateRuleEnabled returns empty object for unknown id', async () => {
+    const rule = await rulesService.updateRuleEnabled(-1, false)
+    expect(rule).toEqual(expect.any(Object))
```

Comment:
This doesn't assert that an empty object was returned, only that an object was returned. I'd suggest something like this as an additional assertion:
```
expect(Object.keys(rule).length).toBe(0)
```
Alternatively, changing the output type for unknown ID to null or undefined (or throwing an error) would arguably give a more clear indication that no rule was found matching the ID.

Reply:
Am returning null now so checking for `toBeNull()`.